main_script.R

```diff
@@ -0,0 +1,113 @@
+# first time run to ensure fonts are working within R markdown
+# install.packages("remotes","extrafont")

```

Comment:
Recommend that you set up 'Renv' (https://rstudio.github.io/renv/articles/renv.html) to help with version control on packages. This has kind of taken over from 'pacrat' and is used to ensure that someone who runs this code in 5 years will use the same R package versions as we have here.

Here is an example for another open source project: https://github.com/RobertASmith/plumberHE/tree/main/renv, not sure if there is something similar in Python.